tsconfig.json

```diff
@@ -1,18 +1,10 @@
 {
   "compilerOptions": {
```

Comment:
Query: why are the comments removed here?

Reply:
Comments are not valid in JSON, however tsconfig is not a "true" JSON file. So we either have to bypass JSON linting for this file or remove comments.

Reply:
It's slightly convoluted, but I have  used this format for comments in package.json - could we use them here too?

"//[ Linting & formatting ]//": "",

Reply:
I've tried something like this before, and the build will error as it doesn't recognise the key. If I move all comments outside "compilerOptions," then it would work. I'd just have to repeat the keys for content.

Reply:
Went with this approach [`d7dfa61` (#40)](https://github.com/ONSdigital/dis-wagtail/pull/40/commits/d7dfa616b5a2b0856ee717803830bb94f63a900b) Thoughts?